zerver/openapi/zulip.yaml

```diff
@@ -10398,6 +10400,13 @@ paths:
                     **Changes**: New in Zulip 6.0 (feature level 146).
                   type: boolean
                   example: true
+                required:
+                  description: |
+                    Whether clients are required to fill this profile field.
```

Comment:
The semantics should be spelled out more -- what is responsible for enforcing it, for example? 

In particular, it should be mentioned that because fields can be added as required fields, clients cannot rely on a `required` field actually being set for every user.

Reply:
I am not entirely sure I understood what needs to be added here. But I have added a paragraph in zulip.yaml:
> As fields can be added as required fields, clients cannot rely on a required
>  field actually set for every user.

Is this OK?

Reply:
I think the following would be good:

> Whether an organization administrator has configured this profile field as required. 
>
> Because the required property is mutable, clients cannot assume that a required custom profile field has a value. The Zulip web application displays a prominent banner to any user who has not set a value for a required field.